plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/FlywayMigration.java

```diff
@@ -25,12 +25,26 @@ private FlywayMigration()
     {
     }
 
+    private static String getLocation(String configDbUrl)
+    {
+        if (configDbUrl.startsWith("jdbc:postgresql")) {
+            return "/db/migration/postgresql";
+        }
+        else if (configDbUrl.startsWith("jdbc:oracle")) {
+            return "/db/migration/oracle";
+        }
+        else if (configDbUrl.startsWith("jdbc:mysql")) {
+            return "/db/migration/mysql";
+        }
+        throw new IllegalArgumentException("Unexpected Url: " + configDbUrl);
```

Comment:
Would be nice to mention that PostgreSQL, MySQL and Oracle are the only supported databases.

IMO the URL validation should be done in `DbResourceGroupConfig` via a `@PostConstruct public void validate() {...}` method (see existing examples in code).